packages/angular/src/generators/move/lib/update-module-name.spec.ts

```diff
@@ -114,118 +128,4 @@ describe('updateModuleName Rule', () => {
       );
     });
   });
-
-  describe('rename', () => {
```

Comment:
Why is this test suite removed? Is it covered anywhere else?

Reply:
I'm not actually sure. I'll double check